cmd/up/up.go

```diff
@@ -354,9 +366,12 @@ func (up *upContext) activate(isRetry, autoDeploy, build bool) error {
 	}()
 
 	prevError := up.waitUntilExitOrInterrupt()
+	if err != nil {
+		log.Infof("command finished with error: %s", prevError)
+	}
 
 	if up.shouldRetry(ctx, prevError) {
-		log.Yellow("\nConnection lost to your development container, reconnecting...\n")
+		log.Yellow("\nConnection lost to your development container, reconnecting...")
```

Comment:
the console still looks wrong after a disconnect. I'm going to try and  fix that as well.